orders_test.go

```diff
@@ -86,6 +86,7 @@ func (ts *TestSuite) TestPlaceOrder(t *testing.T) {
 		Squareoff:         100,
 		Stoploss:          100,
 		TrailingStoploss:  100,
+		AuctionNumber:     7359,
```

Comment:
Shouldn't this be part of a separate test, say `TestPlaceAuctionOrder`?

Reply:
Yeah, maybe, we can add test cases for all different varieties like Iceberg, co, etc.

Reply:
Remove this line?